eth/vm/forks/constantinople/transactions.py

```diff
@@ -0,0 +1,30 @@
+from eth.vm.forks.byzantium.transactions import (
+    ByzantiumTransaction,
+    ByzantiumUnsignedTransaction,
+)
+
+from eth.utils.transactions import (
+    create_transaction_signature,
+)
+
+
+class ConstantinopleTransaction(ByzantiumTransaction):
+    @classmethod
+    def create_unsigned_transaction(cls, nonce, gas_price, gas, to, value, data):
+        return ByzantiumUnsignedTransaction(nonce, gas_price, gas, to, value, data)
```

Comment:
Should probably be `ConstantinopleUnsignedTransaction`

Reply:
:eagle: :eye: